src/docfx/lib/LocalizationUtility.cs

```diff
@@ -65,6 +65,7 @@ public static string AddLeftToRightMarker(CultureInfo culture, string text)
                         return Path.Combine(fallbackRepoPath, docsetSourceFolder);
                     }
                 }
+                throw Errors.System.GitCloneFailed(fallbackRemote, fallbackBranch ?? "").ToException();
```

Comment:
Consider call `TryResolvePackage` for `fallbackBranch` and `ResolvePackage` for `main`. `ResolvePackage` method may throw different exception types or message in the future and it shouldn't be a concern of this method.